cmd/bmp180/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// bmp180 reads the current temperature and pressure from a BMP180.
```

Comment:
/me start thinking about more coherency for package docstring
https://godoc.org/periph.io/x/periph/devices#pkg-subdirectories

At the very least, it shall start with: `Package bmp180 ...`

Include a Reference section with a link to the datasheet.
https://ae-bst.resource.bosch.com/media/_tech/media/datasheets/BST-BMP180-DS000-121.pdf